tests/models/speecht5/test_tokenization_speecht5.py

```diff
@@ -52,12 +52,24 @@ def get_input_output_texts(self, tokenizer):
         output_text = "this is a test"
         return input_text, output_text
 
+    def get_numeric_input_output_texts(self):
+        input_text = "I have $123.45 and owe €59.78. My balance is -₴876.90 and have 73% stocks in my company which equals to ₦72649201"
+        output_text = "I have one hundred and twenty three point four five dollars and owe fifty nine point seven eight euros. My balance is minus eight hundred and seventy six point nine zero ukrainian hryvnia and have seventy three percent stocks in my company which equals to seventy two million six hundred and forty nine thousand two hundred and one nigerian naira"
+        return input_text, output_text
+
     def get_clean_sequence(self, tokenizer, with_prefix_space=False, max_length=20, min_length=5):
         input_text, output_text = self.get_input_output_texts(tokenizer)
         ids = tokenizer.encode(output_text, add_special_tokens=False)
         text = tokenizer.decode(ids, clean_up_tokenization_spaces=False)
         return text, ids
 
+    def test_tokenizer_normalization(self):
+        tokenizer = self.get_tokenizer()
+        input_text, output_text = self.get_numeric_input_output_texts()
+        input_ids = tokenizer.convert_tokens_to_ids(tokenizer.tokenize(input_text))
```

Comment:
Could we make this a bit more clear by setting `normalize=True` for the one we want to normalize?

Is the test also maybe a bit better if we check that the output of the tokenizer is equal to the expected text? E.g.
```python
input_text, expected_text = self.get_numeric_input_output_texts()
input_ids = tokenizer.encode(input_text, normalize=True)
output_text = tokenizer.decode(input_ids, skip_special_tokens=True)
self.assertListEqual(output_text, tokenizer.encode)
```